### PR TITLE
Release v0.2.0

### PR DESCRIPTION
**Pull request by gadomski**

### CODE REVIEW

File: RELEASING.md [hunk 1]
```diff
@@ -7,9 +7,9 @@ Setup:
 Then:
 
 1. Create a new branch: `release/vX.Y.Z`
-2. Update the versions all `Cargo.toml` files, as necessary
+2. Update the versions of all `Cargo.toml` files to match
```
Comment (kylebarron) [inline RELEASING.md:10]: What do you think about putting `version` in the workspace `Cargo.toml` so that we don't have to update all the files manually?

Comment (kylebarron) [reply RELEASING.md:10]: https://github.com/kylebarron/arro3/blob/2d9861ff2c75453f0b65081bdd5b42d28d2d5e13/Cargo.toml#L6-L15

https://github.com/kylebarron/arro3/blob/2d9861ff2c75453f0b65081bdd5b42d28d2d5e13/arro3-core/Cargo.toml#L2-L13

Comment (gadomski) [reply RELEASING.md:10]: Ah nice, I like that. I google-failed on whether I could do that. Will fix.

File: python/Cargo.toml [hunk 2]
```diff
@@ -1,13 +1,13 @@
...
 name = "cql2-python"
-version = "0.2.0"
-authors = ["David Bitner <bitner@dbspatial.com>"]
-edition = "2021"
+version = { workspace = true }
+authors = { workspace = true }
+edition = { workspace = true }
 description = "Parse, validate, and convert Common Query Language (CQL2) text and JSON"
 readme = "README.md"
 homepage = "https://github.com/developmentseed/cql2-rs"
```
Comment (kylebarron) [inline python/Cargo.toml:8]: You can probably DRY the homepage too if you want

**Review by kylebarron — APPROVED** [review]

**Review by bitner — APPROVED** [review]

File: python/Cargo.toml (hunk 2 above)
Comment (gadomski) [reply python/Cargo.toml:8]: Thought about that, but then realized I probably want to correct that later (once the docs are published) to make the Python homepage https://developmentseed.org/cql2-rs/latest/python/ (and probably do the same for the CLI as well).

**Pull request merged**